Domain Services/Health/Expense-v1.1.0.yml

```
@@ -0,0 +1,654 @@
+          example: 1680307200000
+        toPeriod:
+          description: End date of demand period  
+          type: number
+          format: int64
+          example: 1682899199000
+        paymentStatus:
+          $ref: '#/components/schemas/PaymentStatus'
+        status:
+          $ref: '#/components/schemas/Status'
+        payer:
+            $ref: '#/components/schemas/Party'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)